ld.so.conf

```diff
@@ -1,3 +1,6 @@
+# libc default configuration
+/usr/local/lib
```

Comment:
@andyshinn without any specific intention, maybe yes, it looks like default line for all debians/ubuntus and probably other distros. Seems kinda *nix generic to me, IMO.